modules/aws/jumpbox/jumpbox.userdata

```diff
@@ -21,19 +22,41 @@ users:
 
 write_files:
   - content: |
-      #!/bin/sh
-      curl -Lo "/usr/local/bin/tctl" "https://binaries.dl.tetrate.io/public/raw/versions/linux-amd64-${tsb_version}/tctl"
-      chmod +x "/usr/local/bin/tctl"
-      cp /usr/local/bin/tctl /usr/share/nginx/html/
+      #!/bin/bash
       curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
       install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
       ${docker_login}
-      tctl install image-sync \
-        --username "${tsb_image_sync_username}" \
-        --apikey "${tsb_image_sync_apikey}" \
-        --registry "${registry}" \
-        --accept-eula \
-        --parallel
+      export PROJECT_CR="${registry}"
+      # to initialize GCR and use skopeo for TSB images sync below
+      docker pull busybox
+      docker tag busybox $PROJECT_CR/busybox
+      docker push $PROJECT_CR/busybox
```

Comment:
Why is this image needed?

Reply:
skopeo does read a metadata first to push, so as there is no images at GCR - it fails, I have to do a silly push to unblock skopeo for the sync (I don't want to do docker pull/push) I want to keep the jumpbox.userdata the same - no exceptions across clouds - it is easier to maintain that way, so the workaround is primarily for GCR.

Reply:
I'm not sure I understand the issue. Could you detail it a bit more?
Also, given hat we sync images one by one, does it happen if you use `skpoeo copy` instead of `sync`?

Reply:
I have been hitting https://github.com/containers/skopeo/issues/1627 with ```skopeo copy```, the issue with ```skopeo sync``` is GCR or ECR is not being initialized (no containers uploaded), i.e. no metadata to read for skopeo sync when it starts the execution, so skopeo sync fails. Steps to reproduce:

1. Create a new GCP project
2. Try to upload using skopeo sync (the upload will fail)
3. Push the docker image towards GCR under a new GCP project
4. Re-try skopeo sync, the upload will succeed. 

I can use a newer version of skopeo container to move to copy, but I will open an issue to improve this later. Technically we may have other containers at this repo, so for now I will keep it as a workaround.

Reply:
I have opened an issue to track it: https://github.com/smarunich/tetrate-service-bridge-sandbox/issues/144